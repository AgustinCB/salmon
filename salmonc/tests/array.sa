print [1,2,3,4][2];
print [2; 4][3];