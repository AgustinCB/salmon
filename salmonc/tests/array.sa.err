Constants: [Integer(2), Integer(4), Integer(3), Integer(1), String(0)]
Instructions: [Instruction { instruction_type: Constant(0), location: 0 }, Instruction { instruction_type: Constant(1), location: 0 }, Instruction { instruction_type: Constant(2), location: 0 }, Instruction { instruction_type: Constant(0), location: 0 }, Instruction { instruction_type: Constant(3), location: 0 }, Instruction { instruction_type: Constant(1), location: 0 }, Instruction { instruction_type: ArrayAlloc, location: 0 }, Instruction { instruction_type: MultiArraySet, location: 0 }, Instruction { instruction_type: ArrayGet, location: 0 }]
Locations: [Location { address: 0, line: 0 }]
