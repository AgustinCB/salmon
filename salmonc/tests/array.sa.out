0 - Integer(3)
